src/helpers/extractPagesWithAnnotations.js

```diff
@@ -20,7 +20,7 @@ export default async pagesToExtract => {
       annotManager.exportAnnotations({ annotList }).then(xfdfString => {
         doc.extractPages(pagesToExtract, xfdfString).then(data => {
           const arr = new Uint8Array(data);
-          const fileName = 'extractedDocument.pdf';
+          const fileName = doc.filename || 'extractedDocument.pdf';
```

Comment:
Is `filename` exposed in production build? Just want to double check with you.

Reply:
It show up in `readerControl.docViewer.getDocument().filename` but I'm not sure what setting that or if it's reliable so I'll change it